stellar-dotnet-sdk-test/OperationTest.cs

```diff
@@ -946,7 +872,7 @@ public void TestRevokeLedgerEntrySponsorshipOperation()
     public void TestRevokeClaimableBalanceSponsorshipOperation()
     {
         var operation =
-            new RevokeLedgerEntrySponsorshipOperation.Builder("5eUFm5X7itqkrfyAmIjHK7f07NNremw4HpBAJPDQiD0=").Build();
+            new RevokeLedgerEntrySponsorshipOperation.Builder("c8789370cda62cdb5be83642f5af61829c0c77afde7d0fb7577a8f320467563f").Build();
```

Comment:
Not a breaking change since I've only added it this in this PR.
The previous one was a base-64 encoded value, the current one is a hex-encoded value, which is more appropriate, as hex-encoded is what we get using the XDR Viewer.